doc/man1/flow.1

```diff
@@ -180,11 +195,17 @@ Maximum time step size where single precision floating point arithmetic can be u
 \fB\-\-max\-strict\-iter\fR=\fI\,INTEGER\/\fR
 Maximum number of Newton iterations before relaxed tolerances are used for the CNV convergence criterion. Default: 0
 .TP
+\fB\-\-max\-temperature\-change\fR=\fI\,SCALAR\/\fR
+Maximum absolute change of temperature in a single iteration. Default: 5
+.TP
 \fB\-\-max\-welleq\-iter\fR=\fI\,INTEGER\/\fR
 Maximum number of iterations to determine solution the  well equations. Default: 30
 .TP
+\fB\-\-maximum\-number\-of\-well\-switches\fR=\fI\,INTEGER\/\fR
+Maximum number of times a well can switch to the same control. Default: 3
+.TP
 \fB\-\-milu\-variant\fR=\fI\,STRING\/\fR
-Specify which variant of the modified\-ILU preconditioner ought to be used. Possible variants are: ILU (default, plain ILU), MILU_1 (lump diagonal with dropped row entries), MILU_2 (lump diagonal with the sum of the absolute values of the dropped row  entries), MILU_3 (if diagonal is positive add sum of dropped row entrires. Otherwise subtract them), MILU_4 (if diagonal is positive add sum of dropped row entrires. Otherwise do nothing. Default: "ILU"
+Specify which variant of the modified\-ILU preconditioner ought to be used. Possible variants are: ILU (default, plain ILU), MILU_1 (lump diagonal with dropped row entries), MILU_2 (lump diagonal with the sum of the absolute values of the dropped row  entries), MILU_3 (if diagonal is positive add sum of dropped row entrires. Otherwise substract them), MILU_4 (if diagonal is positive add sum of dropped row entrires. Otherwise do nothing. Default: "ILU"
```

Comment:
Adds a misprint ("subtract" gets replaced by "substract").

Reply:
Thanks.
Shoot. These kind of errors drive my crazy. Did not use a flow with the current changes of #3699 for generating the man-page.

Of course I already have built 2021.10-1 packages and uploaded them to [mentors.debian.net](https://mentors.debian.net/package/opm-simulators/):
```
 I typo-in-manual-page

    usr/share/man/man1/flow.1.gz partioning partitioning
    usr/share/man/man1/flow.1.gz substract subtract
```